ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -337,6 +337,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f14
+  cron: 34 16 12,27 * *
+  steps:
+    cluster_profile: azure-qe
```

Comment:
cluster_profile to test azure marketplace image should be `azure-marketplace-qe`